fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/auto/DruidNavigator.java

```diff
@@ -217,5 +208,13 @@ private void queryDruid(SuccessCallback successCallback, String url) {
                 },
                 url
         );
+
+        try {
+            //calling get so we wait until responses are loaded before returning and processing continues
+            responseFuture.get();
+        } catch (InterruptedException | ExecutionException e) {
```

Comment:
All three `TimeoutException`, `InterruptedException` and `ExecutionException` extends exception, I am just wondering why we are catching the latter two while not catching the first one.